File: 백준/Bronze/5988. 홀수일까 짝수일까/홀수일까 짝수일까.c
#include <stdio.h>
#include <string.h>

int main(void) { 
   int a;
   scanf("%d", &a);
   char num[100];

   for(int i = 0; i < a; i++){
      scanf("%s", num);

      int length = strlen(num);
      if((num[length - 1] - '0') % 2 == 0)   printf("even\n");
      else   printf("odd\n");
   }
   
   return 0;
}

File: 백준/Bronze/5988. 홀수일까 짝수일까/홀수일까 짝수일까.cs
using System;

class Program
{
  static void Main(string[] args)
  {
    string input = Console.ReadLine();

    int a = int.Parse(input);

    for(int i = 0; i < a; i++){
      string str = Console.ReadLine();
      int b = int.Parse(str[str.Length - 1].ToString());

      if(b % 2 == 0)  Console.Write("even\n");
      else  Console.Write("odd\n");
    }
  }
}